internal/pkg/template/templates/workloads/partials/cf/autoscaling.yml

```diff
@@ -80,4 +80,147 @@ AutoScalingPolicyECSServiceAverageMemoryUtilization:
       ScaleInCooldown: 120
       ScaleOutCooldown: 60
       TargetValue: {{.Autoscaling.Memory}}
-{{- end}}
\ No newline at end of file
+{{- end}}
+{{- if (and .Subscribe .Autoscaling.QueueDelay) }}
+BacklogPerTaskCalculatorFunction:
```

Comment:
TODO: I should add the CloudWatch log explicitly here and limit the log retention to only 2 days so that we don't accumulate cost for users

Reply:
Done.

Reply:
Is it 3 instead of 2?